api/src/main/resources/defaultAtomfeedConfig.json

```diff
@@ -1,13 +1,169 @@
 [
   {
+    "openMrsClass": "org.openmrs.Location",
+    "enabled": "false",
+    "title": "Location",
+    "category": "location",
+    "linkTemplates": {
+      "rest": "openmrs/ws/rest/v1/location/{uuid}?v=full",
+      "fhir": "openmrs/ws/fhir/v1/location/{uuid}?v=full"
```

Comment:
Link template for REST API is configured properly. However, FHIR API has gently another template. In every resource template should looks like openmrs/ws/fhir/Location/{uuid}
- resource should start with capital Letter ( i.e. Location here)
- v1 and ?v=full should disappear from the template

Only in that case templates give us possibility to get resources from FHIR

Reply:
Ok, I will fix it